EIPS/eip-5732.md

```diff
@@ -1,10 +1,11 @@
 ---
 eip: 5732
-title: Simple Commit Interface
+title: Commit Interface
 description: A simple but general commit interface to support commit-reveal scheme.
 author: Zainan Victor Zhou (@xinbenlv)
 discussions-to: https://ethereum-magicians.org/t/erc-5732-simple-commit-interface-to-support-commit-reveal-schemes/11115
-status: Review
+status: Last Call
+last-call-deadline: 2022-11-01
 type: Standards Track
 category: ERC
 created: 2022-09-29
```

Comment:
On L31-34, the interface has errors. It should be:

```solidity
interface IERC_COMMIT {
    event Commit(bytes32 commitment, bytes extraData);
    function commit(bytes32 commitment, bytes calldata extraData) external;
}
```

`bytes32` instead of `byte32`, and events don't specify calldata.